Homework5/SparseMatrix/ElementDirections.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace SparseMatrix
+{
+    public class ElementDirections
+    {
+        public int row;
```

Comment:
PascalCase for public fields: Row, Columns
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions